src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java

```diff
@@ -123,20 +124,34 @@ public void registerAbilities(List<IFluidTank> abilityList) {
         abilityList.addAll(isExportHatch ? this.exportFluids.getFluidTanks() : this.importFluids.getFluidTanks());
     }
 
-    @Override
-    public void setupNotifiableMetaTileEntity(MetaTileEntity metaTileEntity) {
+    private NotifiableFluidTank getHatchHandler() {
         NotifiableFluidTank handler = null;
-        if (isExportHatch) {
+        if (isExportHatch && getExportFluids().getTankAt(0) instanceof NotifiableFluidTank) {
```

Comment:
Will `getTankAt(0)` work for the upcoming multi-fluid hatches? A question for @DStrand1 about if `getTankAt(0)` will properly track the tanks of different fluids in the multi-fluid hatches.

Reply:
dan has implemented a proper notified tank list on the multi-fluid hatch, after / when its merged we can use that.